src/main.rs

```diff
@@ -146,23 +144,23 @@ fn setup() -> Result<(WindowSize, NeovimRuntime)> {
     // Neovide also includes some other systems which are globally available via lazy static
```

Comment:
nit: I believe the only remaining lazy static is settings? Maybe this can be reworded to not imply that there are multiple

Reply:
I added documentation for the running tracker, which is still remaining.